modules/installation-osp-converting-ignition-resources.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-converting-ignition-resources_{context}"]
+= Preparing the bootstrap Ignition file
+
+The bootstrap Ignition file that the installation program generates is too large to pass to {rh-openstack} directly.
+
+First, upload the bootstrap Ignition configuration file. Then, create a small Ignition file ("shim") that downloads the generated ignition file to Nova as user data.
+
+.Prerequisites
+
+* The bootstrap Ignition file that the installation program generates (`bootstrap.ign`)
+* The infrastructure ID from the installation program's metadata file is set as an environment variable (`$INFRA_ID`)
+** If the variable is not set, see *Creating the Kubernetes manifest and Ignition config files*.
+* An HTTP(S)-accessible way to store the bootstrap ignition file
+** The documented procedure uses Object Storage (Swift), but you could also use Amazon S3, an internal HTTP server, or an ad hoc Nova server
+
+.Procedure
+
+. Edit the bootstrap Ignition file to update the hostname file when it runs. You can do this manually, but it is faster to run the following Python script:
```

Comment:
we should point out somewhere that this is an example using swift object storage service.

Reply:
@maxwelldb You might want to have a look at this freshly merged alternative, which does not use Swift (uses Glance instead, which is supposedly present in 100% of OpenStack installations).

As a side effect, you wouln't have to list the `swift` command-line client as a dependency (uses `openstack` python-openstackclient)

Reply:
(note that the "swift" method you describe is still perfectly valid)

Reply:
Going with Glance. @pierreprinetti Is it right that _if_ Swift is present, the user must/should default to that?

Reply:
> the user must/should default to that
Nope! For this step, Glance will always work, Swift will only work in some cases (i.e. when Swift is available and correctly configured). As a consequence, you might want to remove Swift instructions for the sake of simplicity.

Reply:
@pierreprinetti Excellent news. I'm going to comment out the Swift stuff for now.